aries-site/src/examples/components/tag/TagResource.js

```diff
@@ -37,57 +39,59 @@ const SectionDetails = ({ data }) => (
 );
 
 export const TagResource = () => (
-  <Box width={{ max: 'xxlarge' }} margin="auto" overflow="auto" fill>
-    <Box align="start" pad="medium">
-      <Button label="Devices" icon={<FormPrevious />} />
-    </Box>
-    <Box alignSelf="center" gap="large" pad="medium" flex={false}>
-      <Box
-        border={{ color: 'border-weak', side: 'bottom' }}
-        align="start"
-        direction="row"
-        justify="between"
-        pad={{ bottom: 'medium' }}
-        flex={false}
-      >
-        <Box>
-          <Heading size="small" margin="none">
-            Store C-3P0
-          </Heading>
-          <Text size="large">Aruba 530 32930F</Text>
-        </Box>
-        <Button icon={<More />} />
+  <Page kind="narrow">
+    <PageContent>
+      <Box align="start" pad="medium">
```

Comment:
Wondering if we need this Box at all -- can we have `alignSelf="start"` on the Button itself and remove the Box all together. If we need spacing between the button and the heading can we wrap those in a Box and add gap?

Reply:
We do need the box otherwise the button stretches across the width of the PageContent

Reply:
I think this should work, right?

```
 <Button alignSelf="start" label="Devices" icon={<FormPrevious />} />
```

Reply:
^ Yes that is what I was proposing